### ISSUE TITLE
Test in-app purchase: “android.test.purchased” did not work now

### ISSUE THREAD

**Issue by pokekingvn** [onepf/OpenIAB-Unity-Plugin#98]
i found a sample project with make in-app purchase here, with OpenIAB plugin: https://github.com/GrimReio/OpenIAB-sample-game Few days ago, i had tested the project with Google id like "android.test.purchased" and it successfully show me the fake money to test. I tested again and again, and it seem no bug apears. But just from yesterday and til today, when i tested with that id, it still show "Fake money", but return "Google has stop...", then the purchase became failed. I tested again in 2 other phones, but it returned the same result. I tried with Soomla plugin, but can't fix the problem. Beside that, i switched to iOS platform, and it show in-app purchase well. So does it a error by Google, or Google has changed something with in-app purchase?

**Comment by yashaswi** [comment]
same problem i am facing with my game..please help

**Comment by pokekingvn** [comment]
the only thing i can do now is uninstalled updates of Google Store (new is 6.0.0) to the old (mine is 5.2.13) and i can test perfact again.
But if i don't turn auto update to Google Store, it's going to 6.0.0 and the error back.

**Comment by yashaswi** [comment]
yep,I uninstalled the Update it's worked with out any error..I think there is some problem with googlestore 6.0.0, so what is the solution for that??

**Comment by pokekingvn** [comment]
i'm afraid the only way is waiting fix update from Google.
